src/database/index.tsx

```diff
@@ -112,3 +117,52 @@ export const getSavedState = async (): Promise<EpidemicModelPersistent | null> =
     return null;
   }
 };
+
+const getBaselineScenario = async () => {
+  await authenticate();
+
+  if (!firebase.auth().currentUser) {
+    throw new Error("Firebase user unexpectedly not set");
+  }
+
+  const db = firebase.firestore();
+  const query = db
+    .collection(scenariosCollectionId)
+    .where("baseline", "==", true);
+
+  const results = await query.get();
+  const userId = currrentUserId();
+
+  const baselineScenario = results.docs.find((doc) => {
+    const scenario = doc.data();
+    return userId && scenario.roles[userId] == "owner";
```

Comment:
Ideally there'd be a way to include this filter in the actual query.

Reply:
Making sure I understand: we're querying for _all_ baseline scenarios across all users, and then filtering for ones where the current user has a role and that role is owner. Is that right? `find` returns the first match, right? I wonder if we should perform some assertion that there are not multiple baseline scenarios that match this condition set.